test/integration/datasette_metadata_test.py

```diff
@@ -56,3 +57,17 @@ def test_datasette_metadata_to_yml(ferc1_engine_xbrl):
     for tbl_name in parsed_metadata["databases"]["pudl"]["tables"]:
         if parsed_metadata["databases"]["pudl"]["tables"][tbl_name]["columns"] is None:
             raise AssertionError(f"pudl.{tbl_name}.columns is None")
+
+
+def test_database_metadata(
+    pudl_engine: sa.Engine,
+):
+    """Test to make sure all of the tables in the databases have metadata."""
+    pudl_output = PudlPaths().pudl_output
+    metadata_yml = create_datasette_metadata_yaml()
+    databases = (
+        ["pudl.sqlite"]
+        + sorted(str(p.name) for p in pudl_output.glob("ferc*.sqlite"))
+        + ["censusdp1tract.sqlite"]
```

Comment:
And it's okay to check all of `*.sqlite` because actually internally `check_tables_have_metadata()` has the secret knowledge of which databases we expect to have metadata?

Reply:
Yup! Maybe not the best place to store this information but seems good enough for now.